include/aikido/util/ExecutorThread.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef AIKIDO_UTIL_EXECUTORTHREAD_HPP_
+#define AIKIDO_UTIL_EXECUTORTHREAD_HPP_
+
+#include <atomic>
+#include <chrono>
+#include <functional>
+#include <thread>
+
+namespace aikido {
+namespace util {
+
+/// ExecutorThread is a wrapper of std::thread that calls a callback
+/// periodically.
+///
+/// \code
+/// ExecutorThread exec(
+///     []() { std::cout << "running\n";}, std::chrono::nanoseconds(1));
+///
+/// // thread is running
+///
+/// exec.stop();
```

Comment:
I suggest removing `stop()` from this example code, since `ExecutorThread` is intended to be used primarily as an RAII class. In fact, I'd like to mention the fact that the destructor stops the thread in this comment.